Sources/Common/Store/GlobalDataStore.swift

```diff
@@ -80,14 +38,6 @@ public class CioSharedDataStore: GlobalDataStore {
         self.keyValueStorage = keyValueStorage
     }
 
-    // How to get instance before DI graph is constructed
-    public static func getInstance() -> GlobalDataStore {
-        let newInstance = UserDefaultsSharedKeyValueStorage(
-            deviceMetricsGrabber: DeviceMetricsGrabberImpl()
-        )
-        return CioSharedDataStore(keyValueStorage: newInstance)
-    }
```

Comment:
The `getInstance` function was previously only needed because of `DIGraph` not getting constructed until a `siteId` was provided by customers. 

But, with `DIGraphShared`, we can resolve all dependencies immediately. So, a function like this is not needed. I deleted it so we no longer need to maintain it.